test/check-machines-storage-pools

```diff
@@ -471,6 +470,9 @@ class TestMachinesStoragePools(VirtualMachinesCase):
         dev = self.add_ram_disk(2)
         m.execute(f"parted {dev} mklabel gpt")
 
+        if m.image == "arch":
+            m.execute("mkdir /media/ || true")
```

Comment:
I don't like `|| true` in tests, unless it has a good justification (like opportunistic cleanup). /media is an FHS dir, so it ought to exist. If so, please drop the command; if not, please drop the `|| true`. If there really is a good reason that it *may* exist on our test images, use `mkdir -p`.

Reply:
Ok, seems the Arch `filesystem` package does not include it. As the tests are nondestructive I added the `|| true` so it doesn't fail on the second run.

Reply:
Adding the '-p' will make the command succeed even if the directory exists.

Reply:
I understand, for nond tests. So I think `-p` is fine then.